Resources/Private/Language/locallang.xlf

```diff
@@ -92,6 +92,10 @@
 			<source>Upload new file</source>
 		</trans-unit>
 
+	  <trans-unit id="tx_ats.prepend.select">
```

Comment:
Please fix the indentation here.

Reply:
Hmm .. the indentation is inconsistent most (all?) language files ... It's the same indentation as the following elements and the same indentation as the three elements above. 

IMHO it's better to fix all indentation issues at once.

Reply:
I would take care ... what's your favorite indentation for xlf? There is no definition in .editorconfig ...

Reply:
You are right, we'll normalize this separately.